liquibase/changes/feat_951.sql

```diff
@@ -0,0 +1,11 @@
+CREATE INDEX IF NOT EXISTS idx_feed_osm_location_group_feed_id_stops_count ON feedosmlocationgroup (feed_id, stops_count);
```

Comment:
If we add the order direction to the index, it might increase the performance of the MAX operation
```suggestion
CREATE INDEX IF NOT EXISTS idx_feed_osm_location_group_feed_id_stops_count_desc ON feedosmlocationgroup (feed_id, stops_count DESC);
```
I'm not 100% of the syntax...

Reply:
done! the syntax was correct